democracy_club/apps/core/report_helpers/views.py

```diff
@@ -19,9 +19,7 @@ def get_context_data(self, **kwargs):
         template = Template(self.markdown_content())
         rendered_template = template.render(RequestContext(self.request))
 
-        markdown_doc = markdown_deux.markdown(
-            rendered_template, style="default"
-        )
+        markdown_doc = markdown.markdown(rendered_template, style="default")
```

Comment:
Here's the docs on how to add TOC using this version of Markdown: 

https://python-markdown.github.io/extensions/toc/

Reply:
This means we can get rid of `context["toc"]` and the related line in the template file